spec/feature_flagger/configuration_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+module FeatureFlagger
+  RSpec.describe Configuration do
+    context 'accessors methods' do
+      subject(:configuration) { described_class.new }
+
+      it { expect(configuration).to respond_to(:storage) }
+      it { expect(configuration).to respond_to(:storage=) }
+      it { expect(configuration).to respond_to(:yaml_filepath) }
+      it { expect(configuration).to respond_to(:info) }
+    end
```

Comment:
So, it's better to keep de load of `yaml_filepath` in main module, and then pass the loaded file as arg, and just assign in Configuration, right ?

So, with this change of dependency, Configuration class will just handle the `storage` config, right ?